### PR TITLE
Fixing Tabs & updating tyk_tyk-dashboard.repo conf

### PR DESCRIPTION
**Pull request by asoorm**

### CODE REVIEW

File: tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md [hunk 1]
```diff
@@ -33,57 +33,59 @@ sudo yum install python34
...
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.0/x86_64/
+gpgcheck=0
+enabled=1
 ```
 
 Finally we'll need to update our local cache, so run:
 ```{.copyWrapper}
-    sudo yum -q makecache -y --disablerepo='*' --enablerepo='tyk_tyk-dashboard' info zabbix
+sudo yum -q makecache -y --disablerepo='*' --enablerepo='tyk_tyk-dashboard' info zabbix
```
Comment (excieve) [inline tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md]: I wonder what `info zabbix` means here.

**Comment by excieve** [comment]
Looks good in general, just not sure where `info zabbix` part came from. The same should be done with the pump and gateway docs.

File: tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md]: Anything to do with this? https://www.zabbix.com/product

Comment (marksou) [reply tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md]: Yes, apart from that, looks fine.

**Review by marksou — COMMENTED** [review]
Fine apart from clearing up what the Zabbix ref means.

File: tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md (hunk 1 above)
Comment (excieve) [reply tyk-docs/content/get-started/with-tyk-on-premise/installation/redhat-rhel-centos/dashboard.md]: Looking at `yum -q makecache --help` I don't even see it accepting positional arguments.

**Pull request merged**